ci/checks/style.sh

```diff
@@ -1,15 +1,22 @@
 #!/bin/bash
-# Copyright (c) 2019, NVIDIA CORPORATION.
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
 #####################
 # cuML Style Tester #
 #####################
 
 # Ignore errors and set path
 set +e
-PATH=/conda/bin:$PATH
+export PATH=/conda/bin:/usr/local/cuda/bin:$PATH
 
-# Activate common conda env
+# Activate common conda env and install any dependencies needed
 source activate gdf
+cd $WORKSPACE
+export GIT_DESCRIBE_TAG=`git describe --tags`
+export MINOR_VERSION=`echo $GIT_DESCRIBE_TAG | grep -o -E '([0-9]+\.[0-9]+)'`
+conda install -c rapidsai -c anaconda \
+      "lapack" \
+      "openblas" \
+      "ucx-py=${MINOR_VERSION}"
```

Comment:
this should only be
```
conda install "ucx-py=${MINOR_VERSION}"
```

`lapack` and `openblas` are already installed in the gpuCI containers

Reply:
this fetches the most up-to-date version of ucx-py for the testing (it is already installed in the container but this makes sure it is current)

Reply:
Done. Can you re-review @mike-wendt ?

Reply:
@mike-wendt do you anything else here? Or does this look good to you?